src/Grid.hpp

```diff
@@ -0,0 +1,90 @@
+//
+//  Grid.hpp
+//  Game of Life
+//
+//  Created by Alex Frasca on 01/05/2023.
+//
+#pragma once
+#ifndef Grid_hpp
```

Comment:
Either `#pragma once` or a header guard, not both

Reply:
fixed.